crates/ruff_python_formatter/tests/snapshots/format@notebook_docstring.py.snap

```diff
@@ -33,7 +44,18 @@ source_type                = Ipynb
     This looks like a docstring but is not in a notebook because notebooks can't be imported as a module.
     Ruff should leave it as is
 """
+
```

Comment:
I think it's correct because the source type is a Jupyter Notebook which doesn't have a concept of module-level docstring. So, it's just a string followed by another string.

Reply:
Hello @dhruvmanila, thanks for the review! Shouldn't the formatter keep the two empty lines between the two strings even though the concept of module-level docstring is missing? From what I understood so far, the existing behavior is not correct and should respect the empty lines as is.